### PR TITLE
main/xorg-server: upgrade to 1.20.0

### PR DESCRIPTION
**Pull request by prspkt**
The following ~~(3)~~ (1) package fails to rebuild against the new video ABI:

- `main/xf86-video-s3virge-1.10.7`
~~- `main/xf86-video-savage-2.3.9`~~
~~- `main/xf86-video-sis-0.10.9`~~

Those packages are at their latest version as seen [here](https://www.x.org/releases/individual/driver) and I'm not sure if they will continue to be supported going forward (xorg-video-abi-24 included).

### CODE REVIEW

**Comment by ncopa** [comment]
> Failed to build packages: main/xf86-video-s3virge main/xf86-video-savage main/xf86-video-sis

**Comment by ncopa** [comment]
looks like there are a couple of patches we could grab: https://cgit.freedesktop.org/xorg/driver/xf86-video-savage/

**Comment by ncopa** [comment]
there are some patches to gra for xf86-video sis too: https://cgit.freedesktop.org/xorg/driver/xf86-video-sis/log/

File: main/xf86-video-s3virge/APKBUILD [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-url="http://xorg.freedesktop.org/"
+url="https://xorg.freedesktop.org/"
 arch="all"
 license="MIT"
 subpackages="$pkgname-doc"
 depends=
 makedepends="xorg-server-dev libxi-dev util-macros xorgproto"
-source="http://www.x.org/releases/individual/driver/$pkgname-$pkgver.tar.bz2
+source="https://www.x.org/releases/individual/driver/$pkgname-$pkgver.tar.bz2
+	check-max[HV]Value.patch
```
Comment (ncopa) [inline main/xf86-video-s3virge/APKBUILD]: can you please rename the patch file to not include `[` or `]` chars? I believe its generally a bad idea to include those chars in a file name, just like it is a bad idea to include chars like `?` and `*` in filenames.

Comment (prspkt) [reply main/xf86-video-s3virge/APKBUILD]: Noted and fixed.

File: main/xf86-input-keyboard/APKBUILD [hunk 2]
```diff
@@ -20,13 +20,13 @@ build() {
 		--build=$CBUILD \
 		--host=$CHOST \
 		--prefix=/usr \
```
Comment (ncopa) [inline main/xf86-input-keyboard/APKBUILD]: trailing `\` needs to be removed.

**Comment by ncopa** [comment]
i have fix for it here. will push it.

Good job!

**Pull request closed without merging**